Automattic/woocommerce-payments
# Add paydown timeline item to transaction details

Fixes #3588

#### Changes proposed in this Pull Request

This PR will add a timeline item showing the paydown transaction in the transaction details page for the charges having a portion taken back as a loan payment. Also it'll show the repayment amount in the transaction details overview card and subtract the amount from the net amount displayed there.

This depends on 1583-gh-Automattic/woocommerce-payments-server to get the paydown event into the timeline. So be sure you've checked out on your server (or it is merged already) before you start testing this.

The timeline item will be shown like this: 

![image](https://user-images.githubusercontent.com/3295/152882804-8a0fd15c-12ea-4f14-923f-be74ee7f71e4.png)

Updated parts: 

- The header has "Loan repayment" node, which is subtracted from the net amount.
- There's a new timeline item showing the loan repayment, with a link to the loan filtered transactions page

#### Testing instructions

* All tests should pass
* Check if the overview has the "Loan repayment: ###" text showing the amount of the repayment on the transaction summary card above.
* Check if you see the timeline item, showing the correct values
* Check if you don't see them both on transactions without loan repayments.
* Check if the net amount on the transaction summary card shows the net amount with the loan repayment subtracted.

-------------------


**Post merge**
